src/components/results/recommendations.vue

```diff
@@ -4,7 +4,7 @@
     class="x-list x-list--align-center x-list--padding-07 x-list--padding-bottom"
     :class="{ 'x-list--align-start': $x.device === 'mobile' }"
   >
-    <h1 class="x-title1" :class="{ 'x-padding--left-05': $x.device === 'mobile' }">
+    <h1 class="x-title1" :class="{ 'x-title1-sm x-padding--left-05': $x.device === 'mobile' }">
```

Comment:
What do you think about using here the tailwind device classes, to do it in the same way everywhere?
Something like:

```suggestion
    <h1 class="x-title1 x-title1-sm desktop:x-title1-md" :class="{ 'x-padding--left-05': $x.device === 'mobile' }">
```

Reply:
We haven't discussed what to do about the mobile only classes so I would leave it as is for now